### ISSUE TITLE
GPIO output doesn't appear to be working.

### ISSUE THREAD

**Issue by nkolban** [nebrius/raspi-io#69]
I am using the latest versions of the packages and my Pi 3B kernel reports - 4.4.13-v7+

I am running the following script:

```
"use strict";

var five = require('johnny-five');
var Raspi = require('raspi-io');

var board = new five.Board({
  io: new Raspi()
});

board.on('ready', function() {
  console.log("Board ready!");
  this.pinMode("GPIO17", five.Pin.OUTPUT);
  this.digitalWrite("GPIO17", 0);
});
```

The result of which should set GPIO17 to output with a low value.  As I ran the program, I wasn't seeing the expected results in the electronics so tested further.  Before running, I ran "gpio readall" and for GPIO17 found that it has a "1" value.  I ran the program and then re-ran "gpio readall" and it remains at a "1" level.  Are there any known issues at the current releases?  Is there a mechanism to enable additional diagnostics?

... later ... ... I think I might have a clue.  In the code, we have the notion of a cached "previousWrittenValue" ... and it defaults to 0.  However, if when we start a script, it may be that the actual physical pin has previously been set "1" and hence its ACTUAL signal level is "1".   If we then execute a set of "0" if the default "previousWrittenValue" is 0, won't the code mistakenly believe that the signal level is already at 0 when it isn't?

**Comment by nebrius** [comment]
Sorry for the delay, I've been pretty swamped lately.

Looking into the code further, there must be something else going on. When Raspi IO is initialized, it force writes a 0 to the output at https://github.com/nebrius/raspi-io/blob/master/index.js#L275.

I don't have a way to test at the moment because I'm traveling, but once I can get my hands on some LEDs or a multimeter, I'll check it out.

**Comment by nebrius** [comment]
I finally had a chance to dig into this and found the bug. It's fixed in v7.0.0. Let me know if it works for you!

**Comment by nebrius** [comment]
Closing this issue. If you run into any other issues, let me know!
